examples/Demo_Eigendistortion.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
<p><code>po.load_images</code>  will do what you have <code>to_tensor</code>  do. The following returns the tensor (you'll still have to chop off some to make it square:</p><pre class="ql-syntax" spellcheck="false">image = po.load_images('../data/Parrot.png')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LabForComputationalVision/plenoptic/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>nice. done.</p><p> </p>